Web Experiences/Blueprints/side-bar/side-bar.css

```diff
@@ -1,114 +1,147 @@
+/* accomodate every size screen by default */
```

Comment:
As in alert bar review with spellings and #bx-{{ref}} comment

Reply:
Done